fetch.bs

```diff
@@ -2393,7 +2393,7 @@ with a <i>CORS flag</i> and <i>recursive flag</i>, run these steps:
  <a lt="block bad port">should fetching <var>request</var> be blocked due to a bad port</a>,
  <a href=https://w3c.github.io/webappsec-mixed-content/#should-block-fetch>should fetching <var>request</var> be blocked as mixed content</a>,
  or
- <a href=https://w3c.github.io/webappsec-csp/#should-block-request>should fetching <var>request</var> be blocked by Content Security Policy</a>
+ [[csp-3#should-block-request]]
```

Comment:
Hrm. CSP marks up headers as algorithms:

```
<h3 id="foo" algorithm>
  Foo the bar.
</h3>
```

That syntax doesn't seem to export a definition; it does export the header, which the syntax in this patch links to. I agree that the linking text is a little verbose... But CSP would be (even more!) verbose if I added a distinct `<dfn>` with the same text as the heading.

@tabatkins: is there a better syntax I should be using?